docs/Salesforce-batchsink.md

```diff
@@ -6,7 +6,7 @@ Description
 A batch sink that inserts sObjects into Salesforce.
 Examples of sObjects are opportunities, contacts, accounts, leads, any custom objects, etc.
 
-Plugin can only insert not upsert.
+Currently, only inserts are supported. Upserts are not supported.
```

Comment:
Let's add a part in the design doc about what would be needed to support updates and upserts, and revisit in a future PR.

Reply:
Added to design doc.
Also created a jira for the change
https://issues.cask.co/browse/CDAP-15455